Update instructions about running docker container with videoroom in README.md (#41)
membraneframework/membrane_videoroom: README.md

# Membrane Videoroom
Membrane Videoroom is Open Source project, written in Elixir, for Real-time video and audio communication. 
Membrane Videoroom uses [membrane_rtc_engine](https://github.com/membraneframework/membrane_rtc_engine).

> Link to [demo](https://videoroom.membrane.stream)

Reviewer: The link is duplicated and I think it should have a separate section
```suggestion
## Try it live!
You can test the Videoroom at [https://videoroom.membrane.stream](https://videoroom.membrane.stream)
```

> ## How to use demo as a client
> To see demo of Membrane Videoroom, open this [link](https://videoroom.membrane.stream) (recommended browser is Google Chrome).

Reviewer: ```suggestion
### Using the demo 

The recommended browser for using the demo is Google Chrome.
```

Default environmental variables are available in `.env` file. To run Membrane Videoroom, firstly you have to set up integrated TURNs environments there. Add these 2 lines to this file:
```bash
INTEGRATED_TURN_PORT_RANGE=50000-50050
INTEGRATED_TURN_IP={IPv4 address of one of your network interfaces}
```
`INTEGRATED_TURN_PORT_RANGE` describes the range, where TURN servers will try to open ports. The bigger range is, the more users server will be able to handle. If you will use `--network=host` option when running `docker`, there is no need to set the value of this variable. Otherwise, remember to publish range of ports on `UDP` traffic. 
`INTEGRATED_TURN_IP` is the IP address, on which TURN servers will listen. You can get it from the output of `ifconfig` command. To make the server available from your local network, you can set it to address like `192.168.*.*`.

To build docker image from source, run
```bash
$ docker build  -t membrane_videoroom .
```

To start a container with this image, run 
```bash
$ docker run --network=host --env-file .env membrane_videoroom
```
or
```bash

> $ docker run -p 50000-50050:50000-50050/udp -p 4000:4000/tcp --env-file .env membrane_videoroom

Reviewer: Instead of suggesting editing the `.env` file here, you should rather add `-e` for it

> ***NOTE 1*** There might be a problem with running `--network=host` on `macOS`, so the latter command is recommended on that operating system.
> ***NOTE 2*** Remember to update the range of published ports if setting the `INTEGRATED_TURN_PORT_RANGE` to a value different than in this guide.

Reviewer: ```suggestion
***NOTE 1*** There might be a problem with running `--network=host` on `macOS`, so the latter command is recommended on that operating system.

***NOTE 2*** Remember to update the range of published ports if setting the `INTEGRATED_TURN_PORT_RANGE` to a value different than in this guide.
```

```

***NOTE 1*** There might be a problem with running `--network=host` on `macOS`, so the latter command is recommended on that operating system.
***NOTE 2*** Remember to update the range of published ports if setting the `INTEGRATED_TURN_PORT_RANGE` to a value different than in this guide.

Alternatively, you can also use image from Docker Hub
```bash
$ docker run --network=host --env-file .env membraneframework/demo_webrtc_videoroom_advanced:latest
```
```bash

> $ docker run -p 50000-50050:50000-50050/udp -p 4000:4000/tcp --env-file .env membraneframework/demo_webrtc_videoroom_advanced:latest

Reviewer: I'd reorder these commands - using DockerHub image should be described as the way to go and building from sources should be the alternatives

> 

Reviewer: A few lines above is a broken 
```
`Dockerfile**
```

pls fix that as well

Default environmental variables are available in `.env` file. To run Membrane Videoroom, firstly you have to set up integrated TURNs environments there. Add these 2 lines to this file:
```bash
INTEGRATED_TURN_PORT_RANGE=50000-50050
INTEGRATED_TURN_IP={IPv4 address of one of your network interfaces}
```
`INTEGRATED_TURN_PORT_RANGE` describes the range, where TURN servers will try to open ports. The bigger range is, the more users server will be able to handle. If you will use `--network=host` option when running `docker`, there is no need to set the value of this variable. Otherwise, remember to publish range of ports on `UDP` traffic. 
`INTEGRATED_TURN_IP` is the IP address, on which TURN servers will listen. You can get it from the output of `ifconfig` command. To make the server available from your local network, you can set it to address like `192.168.*.*`.

To build docker image from source, run
```bash
$ docker build  -t membrane_videoroom .
```

To start a container with this image, run 
```bash
$ docker run --network=host --env-file .env membrane_videoroom
```
or
```bash
$ docker run -p 50000-50050:50000-50050/udp -p 4000:4000/tcp --env-file .env membrane_videoroom
```

> ***NOTE*** There might be a problem with running `--network=host` on `Mac OS`, so second command is recommended on this operating system. Remember also, to update range of published ports, if you will set `INTEGRATED_TURN_PORT_RANGE` to different value, than in this guide.

Reviewer: ```suggestion
***NOTE 1*** There might be a problem with running `--network=host` on `macOS`, so the latter command is recommended on that operating system.
***NOTE 2*** Remember to update the range of published ports if setting the `INTEGRATED_TURN_PORT_RANGE` to a value different than in this guide.
```
